gsa/src/web/authorized.js

```diff
@@ -60,12 +62,14 @@ const Authorized = ({children}) => {
     if (hasValue(isAuthenticated)) {
       dispatch(setIsLoggedIn(isAuthenticated));
 
-      if (!isAuthenticated) {
+      if (isAuthenticated) {
+        renewSession();
```

Comment:
I fear this line gets called to often. I am really not sure about that. It really should only be called once.